packages/runtime/container-runtime/src/opLifecycle/opDecompressor.ts

```diff
@@ -174,6 +80,70 @@ export class OpDecompressor {
 
 		return false;
 	}
+
+	public get currentlyUnrolling() {
+		return this.activeBatch;
+	}
+
+	private isSingleMessageBatch = false;
+	public decompressAndStore(message: ISequencedDocumentMessage): void {
```

Comment:
Can you add a brief doc comment here?  When reading the RMP code there's not much context to what "store" might mean - It can be a way someone would begin to learn about how Op Compression/Decompression works